### PR TITLE
OHRM5X-951: Version changed in footer and about modal

### PR DESCRIPTION
**Pull request by VageeshOrangeHRM**
This PR Contains following changes:
1. Product version changed as OrangeHRM OS 5.0-beta in footer and About modal. (Added in config file).

### CODE REVIEW

File: symfony/lib/config/Config.php [hunk 1]
```diff
@@ -30,6 +30,9 @@ class Config
     public const DOCTRINE_PROXY_DIR = 'ohrm_doctrine_proxy_dir';
     public const TEST_DIR= 'ohrm_test_dir';
 
+    public const PRODUCT_NAME = 'OrangeHRM OS';
+    public const PRODUCT_VERSION = 'OrangeHRM OS 5.0-beta';
```
Comment (RajithaKumara) [inline symfony/lib/config/Config.php:34]: for the version `5.0-beta` enough

File: symfony/plugins/orangehrmCorePlugin/Helper/VueControllerHelper.php [hunk 2]
```diff
@@ -136,7 +136,7 @@ public function getContextParams(): array
                 self::CONTEXT_ICON => $contextIcon,
                 self::COPYRIGHT_YEAR => date('Y'),
                 // TODO:: should get from configurations
-                self::PRODUCT_VERSION => '5.0',
+                self::PRODUCT_VERSION => Config::PRODUCT_VERSION,
```
Comment (RajithaKumara) [inline symfony/plugins/orangehrmCorePlugin/Helper/VueControllerHelper.php:139]: add another key for `productName`

File: symfony/plugins/orangehrmCorePlugin/Helper/VueControllerHelper.php [hunk 3]
```diff
@@ -136,7 +136,7 @@ public function getContextParams(): array
                 self::CONTEXT_ICON => $contextIcon,
                 self::COPYRIGHT_YEAR => date('Y'),
                 // TODO:: should get from configurations
```
Comment (RajithaKumara) [inline symfony/plugins/orangehrmCorePlugin/Helper/VueControllerHelper.php:138]: remove this TODO comment

File: symfony/plugins/orangehrmCorePlugin/Api/Rest/AboutOrganizationAPI.php [hunk 4]
```diff
@@ -62,11 +63,11 @@ public function getOne(): EndpointResult
         $employeeParamHolder = new EmployeeSearchFilterParams();
         $employeeParamHolder->setIncludeEmployees("3");
         $organization = $this->getOrganizationService()->getOrganizationGeneralInformation();
-        $organizationName = $organization instanceof Organization ? $organization->getName() : 'OrangeHRM';
+        $organizationName = $organization instanceof Organization ? $organization->getName() : Config::PRODUCT_NAME;
```
Comment (RajithaKumara) [inline symfony/plugins/orangehrmCorePlugin/Api/Rest/AboutOrganizationAPI.php:66]: don't change this. Revert this

File: symfony/plugins/orangehrmCorePlugin/test/fixtures/testcases/AboutOrganizationTestCase.yaml [hunk 5]
```diff
@@ -13,7 +13,7 @@ GetOne:
     query: [ ]
     data:
       companyName: 'Orange HRM'
-      version: '5.0'
+      version: 'OrangeHRM OS 5.0-beta'
       numberOfActiveEmployee: 3
       numberOfPastEmployee: 0
```
Comment (RajithaKumara) [inline symfony/plugins/orangehrmCorePlugin/test/fixtures/testcases/AboutOrganizationTestCase.yaml:18]: want output like this
```yaml
      companyName: 'OrangeHRM'
      productName: 'OrangeHRM OS'
      version: '5.0-beta'
      numberOfActiveEmployee: 3
      numberOfPastEmployee: 0
```
we may have to update this test (only version attribute) in each release?

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

File: symfony/lib/config/Config.php (hunk 1 above)
Comment (VageeshOrangeHRM) [reply symfony/lib/config/Config.php]: done

File: symfony/plugins/orangehrmCorePlugin/Helper/VueControllerHelper.php (hunk 2 above)
Comment (VageeshOrangeHRM) [reply symfony/plugins/orangehrmCorePlugin/Helper/VueControllerHelper.php:139]: done

File: symfony/plugins/orangehrmCorePlugin/Api/Rest/AboutOrganizationAPI.php (hunk 4 above)
Comment (VageeshOrangeHRM) [reply symfony/plugins/orangehrmCorePlugin/Api/Rest/AboutOrganizationAPI.php]: done

File: symfony/plugins/orangehrmCorePlugin/test/fixtures/testcases/AboutOrganizationTestCase.yaml (hunk 5 above)
Comment (VageeshOrangeHRM) [reply symfony/plugins/orangehrmCorePlugin/test/fixtures/testcases/AboutOrganizationTestCase.yaml]: done

File: symfony/plugins/orangehrmCorePlugin/Helper/VueControllerHelper.php (hunk 3 above)
Comment (VageeshOrangeHRM) [reply symfony/plugins/orangehrmCorePlugin/Helper/VueControllerHelper.php]: done

**Review by RajithaKumara — APPROVED** [review]

**Pull request merged**